From dd5a069b52d927073e0ee7fac33d1cd5a0861145 Mon Sep 17 00:00:00 2001
From: Tim Paine <3105306+timkpaine@users.noreply.github.com>
Date: Sat, 13 Jul 2024 23:51:02 -0400
Subject: [PATCH] .

---
 CMakeLists.txt | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d2df665..0260cb9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -250,15 +250,6 @@ function(csp_autogen MODULE_NAME DEST_FILENAME HEADER_NAME_OUTVAR SOURCE_NAME_OU
         set(CSP_AUTOGEN_PYTHONPATH ${PROJECT_BINARY_DIR}/lib:${CMAKE_SOURCE_DIR}:$$PYTHONPATH )
     endif()
 
-    add_custom_command(OUTPUT "${CSP_AUTOTGEN_CPP_OUT}" "${CSP_AUTOTGEN_H_OUT}"
-        COMMAND ${CMAKE_COMMAND} -E env "PYTHONPATH=${CSP_AUTOGEN_PYTHONPATH}" ${Python_EXECUTABLE} ${CSP_AUTOGEN_MODULE_PATH} -m ${MODULE_NAME} -d ${CSP_AUTOGEN_DESTINATION_FOLDER} -o ${DEST_FILENAME} ${CSP_AUTOGEN_EXTRA_ARGS}
-        COMMENT "generating csp c++ types from module ${MODULE_NAME}"
-        DEPENDS mkdir_autogen_${MODULE_NAME} 
-                    ${CMAKE_SOURCE_DIR}/csp/build/csp_autogen.py
-                    ${CMAKE_SOURCE_DIR}/${MODULE_FILENAME}
-                    csptypesimpl
-    )
-
     set(${SOURCE_NAME_OUTVAR} "${CMAKE_CURRENT_BINARY_DIR}/csp_autogen/${DEST_FILENAME}.cpp" PARENT_SCOPE )
     set(${HEADER_NAME_OUTVAR} "${CMAKE_CURRENT_BINARY_DIR}/csp_autogen/${DEST_FILENAME}.h" PARENT_SCOPE )
 endfunction()
-- 
2.44.0

