From 1d638c4d871b4472eb801b93fc26d077d33a32c5 Mon Sep 17 00:00:00 2001
From: t <t@t>
Date: Thu, 30 Apr 2026 07:52:46 +0900
Subject: [PATCH] allow pyarrow 24

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 5aea1f6..c815bd9 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,7 +5,7 @@ requires = [
     # oldest numpy 2 we can
     "numpy>=2,<2.1; python_version < '3.13'",
     "numpy>=2,<2.2",
-    "pyarrow>=15,<24",
+    "pyarrow>=15,<26",
     "ruamel.yaml",
     "scikit-build",
     "setuptools>=69,<74",
@@ -27,7 +27,7 @@ dependencies = [
     "packaging",
     "pandas",
     "psutil",
-    "pyarrow>=15,<24",
+    "pyarrow>=15,<26",
     "pydantic>=2",
     "pytz",
     "ruamel.yaml",
-- 
2.50.1 (Apple Git-155)

